src/enemy/enemy_type.rs

```diff
@@ -36,9 +71,32 @@ impl EnemyType {
                 sequitur.move_pattern_index = (sequitur.move_pattern_index + 1) % CIRCLE.len();
                 rv
             }
+            EnemyType::Polandball(ref mut polska) => {
+                if polska.move_direction == Dir::CRASH {
+                    polska.move_direction = match rand::random::<u8>() % 4 {
+                        0 => Dir::WEST,
```

Comment:
wait i thought you serialized the enum so you wouldn/t have to do a match statement like this here wasn't that the whole point of assigning them the numerical values

Reply:
A) rong account
B) unfortunately rust lets us map from enums to int discriminants but not back 
C) I don't know what you mean by serialized

Reply:
A) unfortunate
B) unfortunate
C) unfortunate